`make full-source-dist USE_BINARYBUILDER=0` is broken due to LLD

We're unable to provide a full source distribution tarball for 1.9.0-alpha1 due to the following error:
```
# Get all the dependencies downloaded
make[1]: Entering directory '/data/ararslan/julia/deps'
make[1]: *** No rule to make target 'get-lld', needed by 'getall'.  Stop.
make[1]: Leaving directory '/data/ararslan/julia/deps'
Makefile:523: recipe for target 'full-source-dist' failed
make: *** [full-source-dist] Error 2
```
Looks like `make -C deps getall` is looking for `get-lld` and failing to find it. I guess a build system piece was missing from https://github.com/JuliaLang/julia/pull/46455.

--- a/deps/llvm.mk
+++ b/deps/llvm.mk
@@ -314,4 +314,5 @@ $(eval $(call bb-install,llvm-tools,LLVM_TOOLS,false,true))
 
 endif # USE_BINARYBUILDER_LLVM
 
+get-lld: get-llvm
 install-lld install-clang install-llvm-tools: install-llvm

